lms/djangoapps/courseware/toggles.py

```diff
@@ -83,6 +83,34 @@
     f'{WAFFLE_FLAG_NAMESPACE}.disable_navigation_sidebar_blocks_caching', __name__
 )
 
+# .. toggle_name: courseware.disable_navigation_sidebar
+# .. toggle_implementation: WaffleFlag
+# .. toggle_default: False
+# .. toggle_description: Disable navi sidebar on Learning MFE
+# .. toggle_use_cases: temporary
+# .. toggle_creation_date: 2024-03-07
+# .. toggle_target_removal_date: None
+# .. toggle_tickets: AXIMST-611
+# .. toggle_warning: None.
+COURSEWARE_MICROFRONTEND_SIDEBAR_DISABLED = CourseWaffleFlag(
+    f'{WAFFLE_FLAG_NAMESPACE}.disable_navigation_sidebar', __name__
+)
+
+# .. toggle_name: courseware.show_default_right_sidebar
+# .. toggle_implementation: WaffleFlag
+# .. toggle_default: False
+# .. toggle_description: If waffle flag disabled
+# Discussions or Notifications sidebar shouldn't be displayed at all on Learning MFE.
+# If waffle flag enabled - Discussions opens always on the pages with discussions,
+# if user is in Audit and course has verified mode - show Notifications.# .. toggle_use_cases: temporary
+# .. toggle_creation_date: 2024-04-28
+# .. toggle_target_removal_date: None
+# .. toggle_tickets: FC-0056
+# .. toggle_warning: None.
+COURSEWARE_SHOW_DEFAULT_RIGHT_SIDEBAR = CourseWaffleFlag(
+    f'{WAFFLE_FLAG_NAMESPACE}.show_default_right_sidebar', __name__
```

Comment:
To be honest, I think assuming this will be on the right is a mistake: it might not be on the right in all circumstances.  We should come up with a more semantic name for it.  If we're calling the other one the navigation sidebar, we could maybe call this one the "notification" one.  Or something even more generic, like "auxiliary" sidebar.  @jmakowski1123, thoughts?

Reply:
Oh, and we should call it something more immediately descriptive like `always_open_auxiliary_sidebar`.

Reply:
Thanks for the suggestions, I agree that generic name is more suitable for this

Reply:
Applied